test/Microsoft.Sbom.Api.Tests/Executors/ExternalDocumentReferenceWriterTest.cs

```diff
@@ -78,8 +78,6 @@ public async Task PassExternalDocumentReferenceInfosChannel_ReturnsJsonDocWithSe
         {
             var root = result.Document.RootElement;
 
-            Assert.IsNotNull(root);
```

Comment:
Why remove this assert?

Reply:
Another case where .NET doesn't believe that `root` can be null, due to the fact that .NET 8 assumes that reference types are non-nullable unless they are declared nullable. In a case where it returns null, the test will throw a `NullReferenceException` and still fail. Not exactly what we would hope, but still OK